pyntcloud/io/ply.py

```diff
@@ -75,7 +75,7 @@ def read_ply(filename):
                 # element mesh
                 if b'list' in line:
 
-                    if b"vertex_indices" in line[-1]:
+                    if b"vertex_indices" in line[-1] or b"vertex_index" in line[-1]:
```

Comment:
Could you add a new sample .ply file that containes this variation (maybe just copy-paste https://github.com/daavoo/pyntcloud/blob/master/tests/data/diamond_ascii.ply) and a corresponding test case to `tests/integration/io/test_from_file.py` ¿?


Maybe something simple, just read the newly added .ply file and check the number of points, like:

https://github.com/daavoo/pyntcloud/blob/master/tests/integration/io/test_from_file.py#L56

Reply:
on it! I think the "correct" way is what you originally had, but then somewhere along the lines a major player accidentally made it `vertex_index` or something and so now generally both are supported.

Reply:
Added it to the unit tests. It seems like some other unit tests are failing (regardless of my commits), but the specific parameterization of the one i added works.

Reply:
Yes. Don't worry about those failing tests, Will be fixed in #253

Reply:
sorry, forgot to push! should be ready to merge now